core/commands/golfcourse/create/create_golfcourse_command_handler.py

```diff
@@ -0,0 +1,43 @@
+from core.commands.golfcourse.create.create_golfcourse_command import CreateGolfcourseCommand
+from core.commands.location.create.create_location_command import CreateLocationCommand
+from core.common.error_messages import ErrorMessage
+from core.common.results import Result
+from core.data_access.models.golfcourse_model import Golfcourse
+from core.data_access.models.location_model import Location
+from core.data_access.repositories.golfcourse_repository import GolfcourseRepository
+from core.common.mediator import RequestHandler
+from core.data_access.repositories.location_repository import LocationRepository
+from core.dtos.golfcourse_dto import GolfcourseDto
+from core.serializers.golfcourse.create_golfcourse_cmd_serializer import CreateGolfcourseCommandSerializer
+
+
+class CreateGolfcourseCommandHandler(RequestHandler[CreateGolfcourseCommand, Result[GolfcourseDto]]):
+    def __init__(self):
+        self.golfcourse_repository = GolfcourseRepository(Golfcourse)
+        self.location_repository = LocationRepository(Location)
+
+    def handle(self, command: CreateGolfcourseCommand) -> Result[GolfcourseDto]:
+
+        if not self.location_repository.location_exists(locationid=command.locationid):
+            return Result.fail(ErrorMessage.not_found(f"Location with id ({command.locationid}) does not exist..."),
+                               status_code=400)
+
+        serializer = CreateGolfcourseCommandSerializer(data={
```

Comment:
Can i not just remove the location_exists? 

 - because the location id is a PrimaryKeyRelatedFueld, so it should cause an invalid serializer if it does not exist??

I Will test

Reply:
Removing the serializer.is_valid() successfully catches a location-reference that doesnt exist: 
![billede](https://github.com/user-attachments/assets/35294547-de6a-46ea-98f9-64b5ca489e60)